README.md

```diff
@@ -29,10 +29,7 @@ Add `hypertrack-sdk-expo` to [`plugins`](https://docs.expo.io/versions/latest/co
         "hypertrack-sdk-expo",
         {
           "publishableKey": "YourPublishableKey", // find this in the HyperTrack dashboard
```

Comment:
I would add the docs link here:
https://hypertrack.com/docs/install-sdk-expo#set-the-publishable-key